helper/identity/templating.go

```diff
@@ -208,7 +282,28 @@ func performTemplating(ns *namespace.Namespace, input string, entity *Entity, gr
 			return "", ErrNoGroupsAttachedToToken
 		}
 		return performGroupsTemplating(strings.TrimPrefix(input, "identity.groups."))
+
+	case strings.HasPrefix(input, "time."):
+		return performTimeTemplating(strings.TrimPrefix(input, "time."))
 	}
 
 	return "", ErrTemplateValueNotFound
 }
+
+func listGroups(groups []*Group, element string) string {
```

Comment:
Same question about returning valid JSON or not

Reply:
Redid this so in JSON mode, you'll get back strings, objects or lists.